CONFIGURATION.md

```diff
@@ -9,8 +9,13 @@ A sample configuration can be found at [config-sample.json](https://github.com/a
 | log_level  | Y        | String | Broker Log Level (DEBUG, INFO, ERROR, FATAL)
 | username   | Y        | String | Broker Auth Username
 | password   | Y        | String | Broker Auth Password
+| master_password_seed | Y        | String | Seed to generate DB instances master passwords
+| broker_name | Y        | String | Unique name to identify by tag in RDS the instances created by this broker
```

Comment:
It's a little bit unclear description. I would change it to sth like "Unique name of RDS broker, used to construct a tag for instance identification"